### ISSUE TITLE
Include link to 1.0.0-beta SemVer spec

### ISSUE THREAD

**Issue by haacked** [semver/semver.org#52]
tl;dr version:

The SemVer 1.0.0 RTM spec was published twice with changes accidentally. In the first version, the dash was not required for pre-release. In the second version, it was.

The solution we came up with is to call the first version `1.0.0-beta` and add a link to that [version of the spec](https://github.com/mojombo/semver.org/commit/a16c419f09b5c5a26320317fcd331264daae16ef) with the `1.0.0-beta` label (just as we do with the RCs).

That way those that implemented that version of the 1.0.0 spec can point out that they also support this particular revision of SemVer.

For the full discussion, check out https://github.com/mojombo/semver/issues/49

**Comment by theory** [comment]
Hey @Haacked, with respect to your request for a PR, I'm not sure I what should be changed in the text of the site, other than perhaps adding a link to the 1.0.0-beta (1.0.0beta? :-P)). Maybe 1.0.0 should mention the use of a hyphen? I wouldn't say it was optional, of course, though it should be optional for parsers that support both 1.0.0-beta and 1.0.0.

**Comment by haacked** [comment]
@theory Yeah, if you look at the site, you'll see there's links to the RCs. I was thinking just do the same for 1.0.0-beta. I can fix it up after if anything needs to be changed. But helping me get a head start is much appreciated.

**Comment by theory** [comment]
Give that a try.

**Comment by haacked** [comment]
Looks like this was completed a while ago. Pro tip, if your commit includes something like "Fixes #52" then when it's merged into the default branch, it'll close the referenced issue.
